### PR TITLE
Add device '412015' from Legrand. DIN power consumption module.

### PR DESCRIPTION
**Pull request by jd1900**
In another pull request I'll add the homeassistant.js for the other repo.

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -4703,6 +4703,29 @@ const converters = {
...
+                payload.power_alarm_active_value = msg.data['61440'];
+                payload.power_alarm_active = (payload.alarm_active_value > 0);
+            }
+            // 0xf001 = 61441
+            if (msg.data.hasOwnProperty('61441')) {
+                payload.power_alarm_enabled = msg.data['61441'];
+            }
+            // 0xf002 = 61442
+            if (msg.data.hasOwnProperty('61442')) {
+                payload.power_alarm_wh_threshold = msg.data['61442'];
```
Comment (Koenkk) [inline converters/fromZigbee.js:4724]: What does `wh` mean?

**Comment by Koenkk** [comment]
Note to self: after this merge https://github.com/Koenkk/zigbee2mqtt/pull/3739

File: converters/fromZigbee.js (hunk 1 above)
Comment (jd1900) [reply converters/fromZigbee.js:4724]: Watts per hour.
1000 wh = 1 Kwh

Comment (jd1900) [reply converters/fromZigbee.js:4724]: In the official app the jumps are not fine grain. The smaller is 1.5 Kwh, and sends to the device 1500 as value. Thus, I name as wh

**Comment by Koenkk** [comment]
Thansk!

**Pull request merged**